windows/src/engine/keyman32/k32_lowlevelkeyboardhook.cpp

```diff
@@ -304,31 +240,33 @@ LRESULT _kmnLowLevelKeyboardProc(
 }
 
 BOOL ProcessHotkey(UINT vkCode, BOOL isUp, DWORD ShiftState) {
-  if (UseRegisterHotkey()) {
-    return FALSE;
-  }
 
   Hotkeys *hotkeys = Hotkeys::Instance();   // I4641
   if (!hotkeys) {
+    SendDebugMessageFormat("Failed to get Instance");
     return FALSE;
   }
 
   Hotkey *hotkey = hotkeys->GetHotkey(ShiftState | vkCode);   // I4641
   if (!hotkey) {
+    SendDebugMessageFormat("GetHotkey Null");
```

Comment:
This would log for every keystroke, probably not needed?

Reply:
All this got cleaned up in the other 2nd PR of the split #12259. But yes the were very much developer logging only.